jni_gen/examples/in_app_java/.metadata

```diff
@@ -0,0 +1,30 @@
+# This file tracks properties of this Flutter project.
```

Comment:
The jni_gen/examples folder needs a README.md with the documentation that is on this PR.

* What both examples do.
* Links to the examples.
* Where the run on.